Tests/ScryfallKitTests/CaseIterableTests.swift

```diff
@@ -0,0 +1,139 @@
+//
+// CaseIterableTests.swift
+//
+
+import XCTest
+import ScryfallKit
+
+/// A test suite to remind maintainers to make sure that all the known cases for
+/// an enum's `allCases` property.
+///
+/// Some of the enums in ScryfallKit have to get updated a lot because WOTC
+/// is constantly playing with card design. To fill the gap between the release
+/// of a new enum case and the release of a supporting ScryfallKit version,
+/// the `unknown(String)` case was introduced.
+///
+/// Unfortunately, adding an associated value to an enum prevents the compiler
+/// from automatically synthesizing the `CaseIterable` conformance.
+///
+/// The manual conformance of `CaseIterable` for the affected types MUST include all cases
+/// _except_ the `unknown(String)` case. Manually providing this conformance introduces the
+/// risk that a new case will be added to one of these types but NOT added to the `allCases`
+/// array. This test suite aims to prevent that via (ab)use of switch exhaustivity. By switching
+/// on an arbitrary enum case in a unit test, the compiler will error out if a new case is added
+/// to the enum but not added to the switch statement in the test. This should hopefully
+/// remind maintainers to keep `allCases` up to date.
+final class CaseIterableTests: XCTestCase {
+  func testFrameEffect() {
+    let stub = Card.FrameEffect.battle
+    let contains = switch stub {
+    case .legendary: Card.FrameEffect.allCases.contains(.legendary)
+    case .miracle: Card.FrameEffect.allCases.contains(.miracle)
+    case .nyxtouched: Card.FrameEffect.allCases.contains(.nyxtouched)
+    case .draft: Card.FrameEffect.allCases.contains(.draft)
+    case .devoid: Card.FrameEffect.allCases.contains(.devoid)
+    case .tombstone: Card.FrameEffect.allCases.contains(.tombstone)
+    case .colorshifted: Card.FrameEffect.allCases.contains(.colorshifted)
+    case .inverted: Card.FrameEffect.allCases.contains(.inverted)
+    case .sunmoondfc: Card.FrameEffect.allCases.contains(.sunmoondfc)
+    case .compasslanddfc: Card.FrameEffect.allCases.contains(.compasslanddfc)
+    case .originpwdfc: Card.FrameEffect.allCases.contains(.originpwdfc)
+    case .mooneldrazidfc: Card.FrameEffect.allCases.contains(.mooneldrazidfc)
+    case .waxingandwaningmoondfc: Card.FrameEffect.allCases.contains(.waxingandwaningmoondfc)
+    case .showcase: Card.FrameEffect.allCases.contains(.showcase)
+    case .extendedart: Card.FrameEffect.allCases.contains(.extendedart)
+    case .companion: Card.FrameEffect.allCases.contains(.companion)
+    case .etched: Card.FrameEffect.allCases.contains(.etched)
+    case .snow: Card.FrameEffect.allCases.contains(.snow)
+    case .lesson: Card.FrameEffect.allCases.contains(.lesson)
+    case .convertdfc: Card.FrameEffect.allCases.contains(.convertdfc)
+    case .fandfc: Card.FrameEffect.allCases.contains(.fandfc)
+    case .battle: Card.FrameEffect.allCases.contains(.battle)
```

Comment:
If I understand this test correctly, I think only `.battle` will be tested?, might want to use fall through to make sure all cases are covered.